test/pummel/test-child-process-spawn-loop.js

```diff
@@ -4,26 +4,26 @@ const assert = require('assert');
 
 const spawn = require('child_process').spawn;
 
-var SIZE = 1000 * 1024;
-var N = 40;
-var finished = false;
+const SIZE = 1000 * 1024;
+const N = 40;
+let finished = false;
 
 function doSpawn(i) {
-  var child = spawn('python', ['-c', 'print ' + SIZE + ' * "C"']);
-  var count = 0;
+  const child = spawn('python', ['-c', 'print ' + SIZE + ' * "C"']);
```

Comment:
@nodejs/testing This test depends on python, although I understand that our build system depends on it. Is that okay to keep it as it is, or refactor it?